xml/System.ServiceModel.Channels/ServiceChannelProxy.xml

```diff
@@ -1,1536 +1,1540 @@
+      <MemberSignature Language="VB.NET" Value="Custom Event Faulted As EventHandler Implements System.ServiceModel.ICommunicationObject" />
+      <MemberSignature Language="F#" Value="member this.System.ServiceModel.ICommunicationObject.Faulted : EventHandler " Usage="member this.System.ServiceModel.ICommunicationObject.Faulted : System.EventHandler " />
+      <MemberType>Event</MemberType>
+      <Implements>
+        <InterfaceMember>E:System.ServiceModel.ICommunicationObject.Faulted</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.EventHandler</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>Occurs when the communication object first enters the faulted state.</summary>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="System.ServiceModel.ICommunicationObject.Open">
+      <MemberSignature Language="C#" Value="void ICommunicationObject.Open ();" />
+      <MemberSignature Language="ILAsm" Value=".method hidebysig newslot virtual instance void System.ServiceModel.ICommunicationObject.Open() cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.ServiceModel.Channels.ServiceChannelProxy.System#ServiceModel#ICommunicationObject#Open" />
+      <MemberSignature Language="VB.NET" Value="Sub Open () Implements ICommunicationObject.Open" />
+      <MemberSignature Language="F#" Value="abstract member System.ServiceModel.ICommunicationObject.Open : unit -&gt; unit&#xA;override this.System.ServiceModel.ICommunicationObject.Open : unit -&gt; unit" Usage="serviceChannelProxy.System.ServiceModel.ICommunicationObject.Open " />
+      <MemberSignature Language="C++ CLI" Value=" virtual void System.ServiceModel.ICommunicationObject.Open() = System::ServiceModel::ICommunicationObject::Open;" />
+      <MemberType>Method</MemberType>
+      <Implements>
+        <InterfaceMember>M:System.ServiceModel.ICommunicationObject.Open</InterfaceMember>
+      </Implements>
+      <AssemblyInfo>
+        <AssemblyName>System.ServiceModel.Primitives</AssemblyName>
+        <AssemblyVersion>6.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.Void</ReturnType>
+      </ReturnValue>
+      <Parameters />
+      <Docs>
+        <summary>Causes a communication object to transition from the created state into the opened state.</summary>
+        <remarks>This member is an explicit interface member implementation. It can be used only when the &lt;xref:System.ServiceModel.Channels.ServiceChannelProxy&gt; instance is cast to an &lt;xref:System.ServiceModel.ICommunicationObject&gt; interface.</remarks>

```

Comment:
```suggestion
        <remarks>This member is an explicit interface member implementation. It can be used only when the <see cref="T:System.ServiceModel.Channels.ServiceChannelProxy"/> instance is cast to an <see cref="T:System.ServiceModel.ICommunicationObject"/> interface.</remarks>

```